HW9/untitled/src/Office.java

```diff
@@ -0,0 +1,17 @@
+public class Office {
+    /* класс Office, в якого буде поле-масив employees,
+    а також методи, які надають контрольований доступ до внутрішнього стану офісу або інших Employee,
+    наприклад Manager зможе мати додаткові методи, які приймають об'єкт типу Employee на вхід і
+    "змушують" робити його якусь роботу, або методи, які дозволяють керувати офісом
+     */
+
+    private Employee[] employees;
+
+    public Employee getEmployees(){
+        for (int i = 0; i< employees.length; i++){
+            return employees[i];
+        }
+        return null;
```

Comment:
you can `return` something only once
so here either only the first Employee will be returned (ignoring the others) or `null`

The better way would be just return a full array or an empty array (in case it's empty)

Returning `null` from a method is usually considered as a bad practice because some other method may call it expecting not null and will try to process the returned value, facing NullPointerException at some unexpected moment